introduction.md

```diff
@@ -4,3 +4,8 @@ My name is Adele, and I like to learn.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?